packages/client/hmi-client/src/services/models/simulation-service.ts

```diff
@@ -0,0 +1,42 @@
+import { logger } from '@/utils/logger';
+import API from '@/api/api';
+
+interface ForecastParametersType {
+	petri: string;
+	initials: { [key: string]: number };
+	tspan: [number, number];
+	params: { [key: string]: number };
+}
```

Comment:
You should use the `@TSModel` annotations in your `/models/simulationservice/SimultionParams.java` to export those automatically within `/types/Types.ts`.

Reply:
will do this in a separate clean up PR